packages/scenario/test/relevant.test.ts

```diff
@@ -407,4 +409,167 @@ describe('Relevance - TriggerableDagTest.java', () => {
 			}
 		);
 	});
+
+	describe('non-relevant node values', () => {
+		/**
+		 * **PORTING NOTES**
+		 *
+		 * - Rephrase? Unclear how best to adapt this particular singular/plural,
```

Comment:
I try to at least hew to the format where I can. In theory, the intent of the format is that it can serve as both test and documentation (which is why BDD-ish tests are often referred to as "specs"). I also think it's often a useful thought exercise when writing (and maintaining) tests, to try to make sure that the test describes the behavior-under-test in accurate and (hopefully) plain language.